test/Test/BaseDAO/FA2.hs

```diff
@@ -126,7 +127,7 @@ validTransferOwnerScenario = uncapsNettest $ do
 
 updatingOperatorScenario :: (Monad m) => NettestImpl m -> m ()
 updatingOperatorScenario = uncapsNettest $ do
-  ((owner1, _), (owner2, _), dao, _) <- originateBaseDAO
+  ((owner1, _), (owner2, _), dao, _) <- originateBaseDAO DAO.defaultConfig
```

Comment:
`originateBaseDAO DAO.defaultConfig` seems to be used in many places, I would define a helper for that. Or hardcode `defaultConfig` in `originateBaseDAO` if we don't use anything else in tests.